.circleci/config.yml

```diff
@@ -166,6 +166,14 @@ jobs:
           name: Run integration tests for the Ivy application
           command: yarn integration:ivy
 
+  integration_ng10_ivy_tests:
+    <<: *job_defaults
+    steps:
+      - *attach_workspace
+      - run:
+          name: Run integration tests for ng10 (Ivy) application
+          command: yarn integration:ng10ivy                
+
```

Comment:
@markwhitfeld i added this line, is that enough?

Reply:
It needs to be added in some other places too.
Search through the file for `integration_ng` to find them ;-)